src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -119,21 +116,11 @@ export const NavBar: React.FC<NavBarPropsType> = ({
 
   const hasUpcomingEvents = futureUpcoming && futureUpcoming.length > 0;
 
-  const hasRadioStations = useCallback(
-    (arr: string[] | undefined): arr is string[] => !isEmpty(arr),
-    []
-  );
-
-  const isSoundCloud =
-    hasRadioStations(radioStations) &&
-    RegExp("soundcloud").test(radioStations[0]);
+  const isSoundCloud = !!firstStation && firstStation.includes("soundcloud");
```

Comment:
```suggestion
  const isSoundCloud = firstStation?.includes("soundcloud");
```

Reply:
[NOTE]
Still not the best test though. Proper would be a RegExp for the specific `soundcloud.com` or similar domain, but this one is good enough for the time being.

Reply:
changed @ 1f8dfd85d82d90d00f2a901db744bd5ba1ea6304